3-networks/envs/shared/main.tf

```diff
@@ -19,6 +19,8 @@ locals {
   env              = "common"
   environment_code = "c"
   bgp_asn_number   = var.enable_partner_interconnect ? "16550" : "64514"
+  default_region1  = "us-west1"
+  default_region2  = "us-central1"
```

Comment:
In the future it would be nice to set this once and pull it through remote state when we switch https://github.com/terraform-google-modules/terraform-example-foundation/issues/685

Reply:
I talked with @daniel-cit regarding adding this to the remote state.